src/DomBuilder/CustomerDirectDebitTransferDomBuilder.php

```diff
@@ -124,6 +124,18 @@ public function visitPaymentInformation(PaymentInformation $paymentInformation):
      */
     public function visitTransferInformation(TransferInformationInterface $transactionInformation): void
     {
+        if (!isset($this->currentPayment)) {
```

Comment:
Do we agree that this introduces a BC break ? If someone is using this already in the wrong order, I think this will break her code right ?
Do you agree that we need to bump a major version here ?

Reply:
When I did some tests, I used the wrong order. If you look at the end of that function, you'll see this line `$this->currentPayment->appendChild(...` at this point an `Uncaught Error: Call to a member function appendChild() on null ...` will occurs. So you may say there's a BC break, but in fact it is a change from an fatal error to a LogicException.

I'd say that it won't have any impact, because if the wrong order is used, the code wouldn't work anyway ;-)

Reply:
Didn't saw that, thanks for clarifying.